packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -655,9 +655,14 @@ export namespace EmrCreateCluster {
      * @default - No allocation strategy, i.e. spot instance type will be chosen based on current price only
      */
     readonly allocationStrategy?: SpotAllocationStrategy;
+
     /**
      * The defined duration for Spot instances (also known as Spot blocks) in minutes.
      *
+     * This is no longer available to new customers from July 1, 2021.
+     * For customers who have previously used the feature, we will continue to support Spot Instances
+     * with a defined duration until December 31, 2022.
+     *
      * @default - No blockDurationMinutes
      */
```

Comment:
```suggestion
     * @default - No blockDurationMinutes
     *
     * @deprecated - Spot Instances with a defined duration (also known as Spot blocks) are no longer available to new customers from July 1, 2021.
     * For customers who have previously used the feature, we will continue to support Spot Instances with a defined duration until December 31, 2022.
     */
```
Should be deprecated.

Reply:
Thanks for the review! I changed it:

https://github.com/aws/aws-cdk/pull/27809/commits/84438170651a27f722ebc28ddfe13309b7bb75e0